### PR TITLE
Add not logic operator

### PR DESCRIPTION
**Pull request by Th3Mouk**
## Purpose
Implement missing operators #60

### CODE REVIEW

File: src/Query/Logic/NotLogic.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+     * @param mixed $value
+     */
+    public function __construct(
+        RethinkInterface $rethink,
+        QueryInterface $query
+    ) {
+        parent::__construct($rethink);
+
+        $this->rethink = $rethink;
+
```
Comment (tbolier) [inline src/Query/Logic/NotLogic.php]: Obsolete empty line

File: src/Query/Logic/NotLogic.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
+<?php
+
```
Comment (marcgeurts) [inline src/Query/Logic/NotLogic.php]: missing declare strict_types

**Comment by tbolier** [comment]
Hey @Th3Mouk 

This PR https://github.com/tbolier/php-rethink-ql/pull/63 fixes the failing unit tests.

Can you process the comments, so we can merge this pull request?

**Comment by tbolier** [comment]
@Th3Mouk I have processed the feedback of @marcgeurts 
@marcgeurts can you approve it?

**Pull request merged**